JS/models/classificationLevels.js

```diff
@@ -1,8 +1,8 @@
-const ClassificationLevel  = {
+const classificationLevels  = {
     UNCLASSIFIED: 'Unclassified',
```

Comment:
The same as for comment regarding Experimantal Types.

Reply:
Done